tests/h2o/test_h2o_model_export.py

```diff
@@ -80,6 +83,23 @@ def test_model_save_load(h2o_iris_model, model_path):
             h2o_model.predict(h2o_iris_model.inference_data).as_data_frame())
 
 
+@pytest.mark.large
+def test_load_model_from_remote_uri_succeeds(h2o_iris_model, model_path, mock_s3_bucket):
```

Comment:
This test case may be doomed (it currently fails) because Moto interferes with requests to non-AWS backends, such as h2o. See https://github.com/spulec/moto/issues/1567.

Reply:
ahh i think it's okay to skip then since the logic is tested in many other places (and there's nothing special for h2o).

Reply:
Got it. Removed the test for now!